src/styles/common.less

```diff
@@ -196,10 +196,10 @@ p {
 }
 
 #helptext > .tooltip-inner {
```

Comment:
This whole section (line 198 to 205) can be gone, since black is the default color.

Reply:
@jgravel  thanks for the review. The requirement didn't include the check mark in the Jira item. I will double check with @jeremy-moffitt . 
Regarding csv link. I asked @jeremy-moffitt if it is ok to do side by side...it looks better that way. @jeremy-moffitt  oked. 
ok will fix the tooltip part.

Reply:
I didnt feel strongly about it, though Julie is correct that the UX mock shows the link underneath, I'd go ahead and put it under if you can

Reply:
@jeremy-moffitt @jgravel  ok will add check mark and move the link button under.